android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/TrackViewHolder.java

```diff
@@ -45,9 +51,16 @@ public void bindTrack(Track track) {
         this.track = track;
 
         int trackColor = Color.parseColor(track.getColor());
+        int sessions = track.getSessions().size();
         String trackName = Utils.checkStringEmpty(track.getName());
 
         trackTitle.setText(trackName);
+        noOfSessions.getBackground().setColorFilter(trackColor, PorterDuff.Mode.SRC_ATOP);
+        if (sessions != 1)
+            noOfSessions.setText(String.format("%d sessions", sessions));
```

Comment:
IMO we should not display `session(s)`. Mostly all the tracks contains more than one sessions so it will be good to display `sessions`. And for track with one session it will be good & meaningful to use  `1 session` instead of `1 session(s)` or `1 sessions`.